### ISSUE TITLE
Add an Accessibility Guide with hoodie-specific examples

### ISSUE THREAD

**Issue by varjmes** [hoodiehq/hoodie-css#1]
As we know, there are a number of ways we can improve the accessibility of Hood.ie. I think we should have a guide to accessibility for contributors: this has the benefit of all new features being accessible (if they aren't, other contributors will now have a guide to make it so) and will also outline how contributors can help make the rest of the current site and all other apps accessible as it is a large task.

Of course, I am happy to do this/help.
I'm not sure where it would go, but it should be as visible as possible.

**Comment by zoepage** [comment]
I really like the idea and would love to add this to http://hoodiehq.github.io/hoodie-css/

**Comment by varjmes** [comment]
Things this guide should include!
- [ ] - What is accessibility (something for everyone, not just people with disabilities)
- [ ] - Examples of issues and how we can cater for them (long term disabilities, short term disabilities, english as a second language, mobile connection etc.)
- [ ] - ARIA Roles
- [ ] - Design (contrast, active & hover states etc.)
- [ ] - Other (skip links, focus states, JavaScript)
